roqoqo-iqm/tests/integration/backend.rs

```diff
@@ -53,21 +77,13 @@ fn run_circuit_single_measurements() {
             backend.run_circuit(&qc).unwrap();
         assert!(bit_registers.contains_key("my_reg1"));
         assert!(bit_registers.contains_key("my_reg2"));
-
-        // NOTE
-        // For now, only the entries of the output registers that are written on are present in the
-        // output registers returned by the backend. In this example, the length of my_reg1 should
-        // be 2 because there are only two single-qubit measurements that actually write on this
-        // register (even though the register is defined by DefinitionBit as having 4 entries).
-        // This could change when we introduce pairings between qubit numbers and positions in the
-        // output registers in the backend.
-        let out_reg = bit_registers.get("my_reg1").unwrap();
-        assert_eq!(out_reg[0].len(), 2);
-        let out_reg = bit_registers.get("my_reg2").unwrap();
-        assert_eq!(out_reg[0].len(), 1);
+    } else {
+        panic!("no IQM_TOKENS_FILE env var found.")
     }
 }
 
+// Test a deterministic circuit with repeated measurements
+// Ignore because the Demo backend returns pseudorandom results and does not implement a simulator
 #[test]
 #[ignore]
```

Comment:
Is this being ignored temporarily, and the demo backend will be replaced later? I would recommend checking that the `out_reg` is not empty and leave it at that, if the results are pseudo-random, instead of skipping the entire test